config/autoload/migrations-db.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+return [
+    'url' =>
+        'mysql://passwordcockpit_db_usr:password@10.0.3.150:3306/passwordcockpit'
```

Comment:
Modify the hardcoded IP

Reply:
The correct url: `        'mysql://passwordcockpit_db_usr:password@passwordcockpit_database/passwordcockpit'
`